frontend/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import './SearchBar.css';
+
+// All this is copied from MainSearch, needs to be changed
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchTerm: '',
+      error: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ searchTerm: event.target.value });
+  }
+
+  handleValidation() {
+    let error = '';
+    let isValid = true;
+    const { searchTerm } = this.state;
```

Comment:
NP: I like to have `const`s precede `let`s to preserve alphabetical order. I also like to have newlines both before and after `if` statements.